src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div class="calendar calendar--start-day--sun calendar--month-length--31">
+        <div class="calendar__day day"></div>
```

Comment:
No need to add a `day` block. Remove it everywhere, please 
```suggestion
        <div class="calendar__day"></div>
```